### PR TITLE
refactor(office): rename to about-us

### LINKED ISSUE #259: Office -> About us
We are missing some important content on the page: who are we and how do we work. I think that can be one page where we can show a little bit about the office but mainly focus on the team: `About us` instead of `Office`.

I would really like to try to make this happen. Let's talk.

### PR DESCRIPTION
**Pull request by milljoniaer**
### What is included?

- Rename Gatsby Page
- rename all the Page Building files
- rename translation keys
- rename rename translation values
- rename Navigation

---

closes #259

### CODE REVIEW

File: src/locales/de/translations.json
Comment (georgiee) [inline src/locales/de/translations.json]: I'm not happy about the dash but I wonder if an underscore would help much here. Technically fine so thanks!

**Review by georgiee — APPROVED** [review]
Please rename the stories as well:

`src/page-building/about-us/office.stories.mdx` -> src/page-building/about-us/about_us.stories.mdx` or so?

Also the story name should be "About us" not "office"
Screen Shot 2021-12-13 at 18 26 25


I already approve, mayeb you can change that.

File: src/locales/en/translations.json
Comment (fischeversenker) [inline src/locales/en/translations.json]: I know this is not necessarily related to this PR but these translations seem fairly "holprig". I suggest "This is who we are" instead of "That is who we are" as well as "Are you interested?" instead of "Did you get interested?":
```suggestion
  "about-us.team.heading": "This is who we are",
  "about-us.team.text": "Our experienced team of engineers and designers",
  "about-us.leadbox.title": "Are you interested?",
```

Some background:
- "That" implies distance. "This is my body, that is your body". "This" is close, "that" is distant. In this case the images of the team are right below the text so there is no distance thus we should be using "this".
- "Did you get interested?" is Denglisch. "Are you interested?" is the proper translation in my opinion.

**Review by fischeversenker — APPROVED** [review]
Thanks for this! :pray:

Everything looks great but I noticed a few janky translations, please see my comments. These translations are not related to this PR so they don't block this PR. I'm approving this but I'd appreciate it if you considered my suggestions 👍

File: src/locales/en/translations.json
Comment (georgiee) [reply src/locales/en/translations.json]: thanks for the catches. right now DE is the primary language and we plan to send the texts to a translator once the german version is approved/ready. But I still would like to fix things like this so I very appreciate your suggestion 👍 The translations could be weird because of deepl and myself looking over every translation but only to catche absolute non-sense. I guess this slipped.

**Pull request merged**
